cmd/testdata/TestCreateCluster_four_partial_deposits_files.golden

```diff
@@ -5,18 +5,30 @@
  "node3",
  "node0/charon-enr-private-key",
  "node0/cluster-lock.json",
- "node0/deposit-data.json",
+ "node0/deposit-data-0-8eth.json",
+ "node0/deposit-data-1-8eth.json",
+ "node0/deposit-data-2-8eth.json",
+ "node0/deposit-data-3-8eth.json",
```

Comment:
Is there a reason to create duplicates files here? Shouldn't the be all the same? We get people to specify the amounts in `deposit_amounts` mostly as a safety to make sure they don't create too few deposits, but in my opinion we should only do one deposit signing process and output one deposit file for each unique amount, as they are deterministic and identical copies. Otherwise I fear people will have to sweat more about activating exactly the correct files when it doesn't matter in practice.

Reply:
Implemented as suggested.